deployment/chainloop/templates/_helpers.tpl

```diff
@@ -82,7 +82,7 @@ awsSecretManager:
 gcpSecretManager:
   secretPrefix: {{ required "secret prefix required" .secretPrefix | quote }}
   projectId: {{ required "project id required" .gcpSecretManager.projectId | quote }}
-  authKey: {{ required "auth key required" .gcpSecretManager.authKey | quote }}
```

Comment:
I think we should still keep the `required` stanza to make sure you provide it and we do not create an empty secret no?

in other words, if we have the required here and we do not properly set the key, deployment will fail at template time, not at runtime. 

What do you think?

Reply:
I think that is a good idea - that would allow to fail fast and correct configuration mistakes.

Reply:
Actually, is it possible to use the `required` function here? The required function accepts an error message and a path to a key. In this case, we just want to check the key has not an empty value, but we want to enforce a static string for the value. Is there a way to do this?

Reply:
You are right, I missed that, `required` does not make sense anymore. 

Maybe we can append something like?

```
{{- if eq .gcpSecretManager.serviceAccountKey "" }}
{{- fail ".Values.gcpSecretManager.serviceAccountKey not set" }}
{{- end }}
```